tests/ci/spec_to_test_mapping.py

```diff
@@ -198,6 +198,11 @@
         '',
         ''
     ],
+    'components/perf-tools/dimemas/SPECS/dimemas.spec': [
+        'dimemas',
+        '',
+        'lmod-defaults-gnu12-openmpi4-ohpc'
```

Comment:
Why only the `openmpi4` flavor is needed here ?
The tests are executed also for `mpich`. And they pass without the dependency.

Reply:
Frankly, I referenced:

https://github.com/openhpc/ohpc/blob/222d7931e7df61b1aeacc4e975e2b859dca3af82/tests/ci/spec_to_test_mapping.py#L149

I also remember I resolved the  `The following module(s) are unknown: "ohpc"` by installing lmod-defaults-gnu12-openmpi4-ohpc. (Give a installs deafult)

Reply:
> Why only the `openmpi4` flavor is needed here ?
> The tests are executed also for `mpich`. And they pass without the dependency.

I think this is necessary if there is a `module load ohpc` somewhere. There is the possibility to set a default compiler MPI combination with the help of the defaults package.

I am not sure why it is loaded in the test, but to solve the errors during `module load ohpc` one of the defaults needs to be installed.

That is also how I worked around this.